ai2thor/server.py

```diff
@@ -27,6 +27,8 @@
 import werkzeug.http
 import numpy as np
 
+from enum import Enum
```

Comment:
Is there a python 2 equivalent? enum isn't available in python 2.7.

Reply:
ok will look for that, we already are using `Enum` in `interact.py`